tutorials/vpn-interop-guides/interop-guide-template/index.md

```diff
@@ -0,0 +1,1003 @@
+```
+
+1. Reserve a static IP address in the VPC network and region where you created the VPN gateway. Make a note of the created address for use in future steps.  
+  
+`gcloud compute --project vpn-guide addresses create \ `
+
+`        --region us-east1 vpn-scale-test-static-ip`  
+
+1. Create a forwarding rule that forwards ESP, IKE and NAT-T traffic toward the Cloud VPN gateway. Use the static IP address `vpn-scale-test-static-ip` you reserved earlier. This step generates a forwarding rule named `fr-esp,` `fr-udp500`, `fr-udp4500` resp.  
+  
+ `gcloud compute --project vpn-guide forwarding-rules create fr-esp \`
+
+```
+        --region us-east1 --ip-protocol ESP --address 35.185.3.177
+          --target-vpn-gateway vpn-scale-test-<vendor-name>-gw-1
+
+            gcloud compute --project vpn-guide forwarding-rules create fr-udp500 \ 
+        --region us-east1 \
+        --ip-protocol UDP --ports 500 --address 35.185.3.177 \
+        --target-vpn-gateway vpn-scale-test-<vendor-name>-gw-1
+
+      gcloud compute --project vpn-guide forwarding-rules create fr-udp4500 \ 
+        --region us-east1 \
+        --ip-protocol UDP --ports 4500 --address 35.185.3.177 \ 
+```
+
+`        --target-vpn-gateway vpn-scale-test-<vendor-name>-gw-1`  
+
+1. Create [Cloud Router](https://cloud.google.com/compute/docs/cloudrouter) with the following command:
+```
+gcloud compute --project vpn-guide routers create vpn-scale-test-<vendor-name>-rtr --region us-east1 \
+  --network vpn-vendor-test-network --asn 65002
+```
+
+1. Create a VPN tunnel on the Cloud VPN Gateway that points toward the external  
+    IP address `[CUST_GW_EXT_IP]` of your on-premises VPN gateway. 
+   1. Supply the shared secret. 
+   1. Set the IKE version. The following example sets the IKE version to 2, which is the default, preferred IKE version. If you need to set it to 1, use `--ike_version 1`. 
+   1. After you run this command, resources are allocated for this VPN tunnel, but it is not yet passing traffic.
+
+        `gcloud compute --project vpn-guide vpn-tunnels create vpn-scale-test-tunnel1 \`
+
+```
+      --peer-address 204.237.220.4 \
+      --region us-east1 --ike-version 2 \
+      --shared-secret MySharedSecret \
+      --target-vpn-gateway vpn-scale-test-<vendor-name>-gw-1 \ 
+```
+
+`      --router vpn-scale-test-<vendor-name>-rtr`  
+
+1. Update the Cloud Router config to add a virtual interface (`--interface-name`) for the BGP peer. 
+-  The BGP interface IP address must be a (https://wikipedia.org/wiki/Link-local_address)** IP address belonging to the IP address range `169.254.0.0/16` and it must belong to same subnet as the interface address of the peer router. The recommended netmask length is `30`. 
+-  Make sure each tunnel has a unique pair of IP addresses. Alternatively, you can leave `--ip-address` and `--mask-length blank, and leave `--peer-ip-address` blank in the next step, and the addresses will be automatically generated for you.  
```

Comment:
check the tick marks.

Reply:
Think you want to change the line to:
`--ip-address` and `--mask-length` blank, and leave `--peer-ip-address` blank